### PR TITLE
[docs]add parameter for helm init cluster

### PR DESCRIPTION
**Pull request by yufan022**
### Motivation
Failed to deploy the pulsar cluster on k8s according to the [document](http://pulsar.apache.org/docs/en/kubernetes-helm/#step-2-use-pulsar-admin-to-create-pulsar-tenantsnamespacestopics).

I checked the pulsar-helm [repository](https://github.com/apache/pulsar-helm-chart), need add parameter `--set initialize=true` to initialize the cluster metadata.

So maybe the document needs to be updated

### CODE REVIEW

**Review by sijie — APPROVED** [review]

File: site2/website/versioned_docs/version-2.7.1/getting-started-helm.md [hunk 1]
```diff
@@ -81,9 +81,14 @@ We use [Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/) i
 
 3. Use the Pulsar Helm chart to install a Pulsar cluster to Kubernetes.
 
+   > NOTE: Please specify `--set initialize=true` when installing a release at the first time. `initialize=true` will start initialize jobs
+   >       to initialize the cluster metadata for both bookkeeper and pulsar clusters.
```
Comment (Jennifer88huang) [inline site2/website/versioned_docs/version-2.7.1/getting-started-helm.md:85]: > **NOTE**  
> You need to specify `--set initialize=true` when installing Pulsar the first time. Setting `initialize=true` starts initializing jobs to initialize the cluster metadata for both BookKeeper and Pulsar clusters.

Comment (Jennifer88huang) [reply site2/website/versioned_docs/version-2.7.1/getting-started-helm.md:85]: When you write "starts initializing jobs to initialize", do you mean "Setting `initialize=true` starts initializing the cluster metadata....

**Comment by Jennifer88huang** [comment]
@yufan022 Thank you very much for your contribution. Does the change apply only to 2.7.1?

File: site2/website/versioned_docs/version-2.7.1/getting-started-helm.md (hunk 1 above)
Comment (yufan022) [reply site2/website/versioned_docs/version-2.7.1/getting-started-helm.md:85]: Reference from https://github.com/apache/pulsar-helm-chart
image

**Comment by yufan022** [comment]
> @yufan022 Thank you very much for your contribution. Does the change apply only to 2.7.1?

@Jennifer88huang Maybe more versions,  It depends on the version of Helm.

File: site2/website/versioned_docs/version-2.7.1/getting-started-helm.md (hunk 1 above)
Comment (Jennifer88huang) [reply site2/website/versioned_docs/version-2.7.1/getting-started-helm.md:85]: @yufan022 Thanks for your reference. I think "This command installs and starts Apache Pulsar" is clear and concise. Maybe it's better to use this one. If you want to add BK, you can refine it as "This command initializes the metadata for both BookKeeper and Pulsar clusters."

**Comment by Jennifer88huang** [comment]
@@yufan022 Thanks for your info. From this commit "apache/pulsar-helm-chart@b44b523", we can see that it affects versions since 2.6.1. Could you please apply the changes to 2.6.1, 2.6.2, 2.6.3, 2.7.0 and the main branch as well? Thank you.

File: site2/website/versioned_docs/version-2.7.1/getting-started-helm.md (hunk 1 above)
Comment (yufan022) [reply site2/website/versioned_docs/version-2.7.1/getting-started-helm.md:85]: You mean change 
`NOTE: Please specify --set initialize=true when installing a release at the first time. `initialize=true` will start initialize jobs to initialize the cluster metadata for both bookkeeper and pulsar clusters.`
to
`This command installs and starts Apache Pulsar` ？

Comment (Jennifer88huang) [reply site2/website/versioned_docs/version-2.7.1/getting-started-helm.md:85]: I mean you can change it into 
```
> **NOTE**  
> You need to specify `--set initialize=true` when installing Pulsar the first time. This command installs and starts Apache Pulsar.
```
or
```
> **NOTE**  
> You need to specify `--set initialize=true` when installing Pulsar the first time. This command initializes the metadata for both BookKeeper and Pulsar clusters.
```

**Comment by yufan022** [comment]
@Jennifer88huang Hi, some new commits. Please take a look.

**Comment by Jennifer88huang** [comment]
@yufan022 Great, thank you. The file in main branch is this file https://github.com/apache/pulsar/blob/master/site2/docs/getting-started-helm.md
Could you apply the changes to this file as well? So it will apply to the next major release (2.8.0). Thank you very much for doing so.

**Comment by yufan022** [comment]
> @yufan022 Great, thank you. The file in main branch is this file https://github.com/apache/pulsar/blob/master/site2/docs/getting-started-helm.md
> Could you apply the changes to this file as well? So it will apply to the next major release (2.8.0). Thank you very much for doing so.

Completed PTAL

**Review by Jennifer88huang — APPROVED** [review]

**Comment by Jennifer88huang** [comment]
@yufan022 Thank you very much for your contribution~

**Pull request merged**